docs/.vitepress/config/en.ts

```diff
@@ -79,6 +79,6 @@ export const enConfig = async () => ({
         icon: 'FAQandOthers',
       },
     ],
-    copyright: '©Copyright 2023 Greptime Inc. All Rights Reserved',
```

Comment:
We may want `2023-2024` or even `2022-2024` according to the company's established day.

A JS script can help like [this](https://github.com/apache/incubator-opendal/blob/d444c48cb5d0bc3015ff73939cedf7545e61800f/website/docusaurus.config.js#L216C579-L216C623):

```js
`Copyright © 2022-${new Date().getFullYear()}`
```

Reply:
cc @alili @Aganivi